composer.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "php/web-php",
+    "description": "The www.php.net site.",
+    "license": "proprietary",
+    "type": "project",
+    "homepage": "https://www.php.net/",
+    "support": {
+        "source": "https://github.com/php/web-php"
+    },
+    "require": {
+        "php": "^7.3"
```

Shouldn't we claim compatibility with PHP 8, e.g.
```suggestion
        "php": "^7.3 || ^8"
```

---

@cmb69 

I would agree if this were a library - but since this is an application, it will only run in a specific production environment, will it not? That specific production environment currently uses PHP 7.3.19, as you reported.

---

I was afraid it would not be possible to use PHP 8 locally, but apparently that is not really an issue due to `platform: php: 7.3.19`. So we can leave this as is.